recipes/dev/early_exit_finetune_distributed.py

```diff
@@ -357,6 +357,10 @@ def setup(self, cfg: DictConfig) -> None:
         if self._loss_fn.__class__.__name__ == "CEWithChunkedOutputLoss":
             # set num_output_chunks for model
             self._model.set_num_output_chunks(self._loss_fn.num_output_chunks)
+        elif getattr(self._loss_fn, "linear_loss", False):
+            raise ValueError(
+                "Linear losses are not supported yet for KD. Please use the deprecated CEWithChunkedOutputLoss."
+            )
```

Comment:
Wrong error message? Also can we open a high-priority issue for this? I don't like being in a state where half our configs are on a deprecated API